hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystemMetrics.java

```diff
@@ -155,6 +153,12 @@ private void testOzoneFileCommit(TestOps op) throws Exception {
     long numKeysAfterDelete = cluster
         .getOzoneManager().getMetrics().getNumKeys();
     Assert.assertTrue(numKeysAfterDelete >= 0);
-    Assert.assertEquals(numKeysBeforeCreate, numKeysAfterDelete);
+    // The intermediate directories in the filePath are added to the directory
+    // table, and are added to the metrics as a key.
+    // When we delete the Key/File/Directory - the intermediate directories
+    // added to the directory table are cleared out using
+    // DirectoryDeletingService - which does not decrement the numKey metrics.
+    // Therefore, the numKeys metrics should be one more than the initial value.
+    Assert.assertEquals(numKeysBeforeCreate + 1, numKeysAfterDelete);
```

Comment:
@rakeshadr 
This looks like a behavior change in the test, could you review this?